pebbles/views/blueprints.py

```diff
@@ -132,6 +137,33 @@ def put(self, blueprint_id):
         db.session.add(blueprint)
         db.session.commit()
 
+    @auth.login_required
+    @requires_group_owner_or_admin
+    def patch(self, blueprint_id):
+        args = self.parser.parse_args()
+        blueprint = Blueprint.query.filter_by(id=blueprint_id).first()
+        if not blueprint:
+            abort(404)
+
+        if args.get('current_status'):
+            blueprint.current_status = args['current_status']
+            db.session.commit()
```

Comment:
db.session.add(blueprint) missing

Reply:
Yes its modifying the fields. db.session.commit() should take care of updating.